docs/entry_points/README.md

```diff
@@ -0,0 +1,5 @@
+# Entry points
+
+The `TaskEntryPoint` is a fundamental class that can be used when developing a Python module to be executed via an entry point in a Python wheel.
+
+When inheriting from this class we ensure the implementation of the `task` method such that these can be easily discovered and manipulated throughout the package.
```

Comment:
add an example of how to use it / how it would look like when inherit from it

Reply:
Added